### PR TITLE
small typos in doc/INSTALL.md

### PR DESCRIPTION
**Pull request by darosior**
A double `git` and a `mode`.

Changelog-None

### CODE REVIEW

File: doc/INSTALL.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For actually doing development and running the tests, you will also need:
 * pip3: to install python-bitcoinlib
 * valgrind: for extra debugging checks
 
-You will also need a version of bitcoind with segregated witness and `estimatesmartfee` economical node, such as the 0.16 or above.
+You will also need a version of bitcoind with segregated witness and `estimatesmartfee` economical mode, such as the 0.16 or above.
```
Comment (ZmnSCPxj) [inline doc/INSTALL.md]: "economical mode" still does not quite make sense to me.... maybe just say "and `estimatesmartfee` implementation"?

Comment (darosior) [reply doc/INSTALL.md]: The economical mode was added later. Maybe :
> and `estimatesmartfee` with economical mode
?

Comment (kristapsk) [reply doc/INSTALL.md]: "`estimatesmartfee` with "ECONOMICAL" fee estimate mode support"? (that "ECONOMICAL" in call caps and quotes as it is how it is called in `bitcoin-cli help estimatesmartfee` output). Maybe also could add that it's Bitcoin Core v0.15.0 or newer.

Comment (darosior) [reply doc/INSTALL.md]: It's specified 0.16 just after and that's 0.16 IIRC ? Like 0.15 `estimatesmartfee` and 0.16 `ECO` mode ?

Comment (darosior) [reply doc/INSTALL.md]: Ah the version was wrong too ! Thanks

Comment (kristapsk) [reply doc/INSTALL.md]: Yes, it was introduced in 0.15. https://bitcoin.org/en/release/v0.15.0#changes-to-fee-estimate-rpcs

**Comment by cdecker** [comment]
ACK https://github.com/ElementsProject/lightning/pull/3432/commits/36bdf394f3fb194aa789c07c9aa1a3fb05520d6c

**Pull request merged**